pymatviz/ptable.py

```diff
@@ -793,6 +796,9 @@ def tick_fmt(val: float, _pos: int) -> str:
     ) | (text_style or {})
 
     for symbol, row, column, *_ in df_ptable.itertuples():
+        if hide_f_block and (row in (6, 7)):
```

Comment:
I thought I would do this after refactor all other plotters with the new `PTableProjector`, but thanks for fixing this.